### ISSUE TITLE
Problem with peer-dependencies [v0.5.0-rc.7]

### ISSUE THREAD

**Issue by andersbc** [falcondev-oss/trpc-vue-query#7]
I am having trouble installing the package because `package.json` requires a non-existing versions 11 of  `@trpc/client` and `@trpc/server` 

```json
  "peerDependencies": {
    "@tanstack/vue-query": "^5.22.2",
    "@trpc/client": "11",
    "@trpc/server": "11",
    "vue": "^3.4.19"
  },
```

Could it be changed to?

```json
  "@trpc/client": "^11.0.0-next || ^11.0.0",
  "@trpc/server": "^11.0.0-next || ^11.0.0",
```

Thanks :-)

**Comment by DrJume** [comment]
Thank you for bringing this to our attention!

The "11" works only when verifying the version when it is already installed. When fresh installing, it tries to install `@trpc/server@11` which does not exist. trpc uses only `next` as the version tag for release candidates.

**Comment by DrJume** [comment]
Should be fixed in `v0.5.0-rc.8`
